src/components/SimpleTable.vue

```diff
@@ -0,0 +1,209 @@
+<template>
+  <section class="aph-simple-table">
+    <table class="table-wrapper">
+      <thead>
+        <tr class="table-header-row">
+          <th v-for="key in columns"
+            @click="sortBy(key)"
+            class="header-cell"
+            :class="{ active: sortKey === key }">
+            {{ key | capitalize }}
```

Comment:
Does this just uppercase the text? Why not use css?

Reply:
Yeah. We can def use css. The use of filters seems pretty cool though!